polymorphic/formsets/utils.py

```diff
@@ -1,11 +1,17 @@
 """
 Internal utils
 """
+import django
 
 
 def add_media(dest, media):
     """
     Optimized version of django.forms.Media.__add__() that doesn't create new objects.
+
+    Only required for Django < 2.0
     """
-    dest.add_css(media._css)
-    dest.add_js(media._js)
+    if django.VERSION >= (2, 0):
+        dest += media
```

Comment:
This will break. The Django code for `dest +=` does the following:

```
    def __add__(self, other):
        combined = Media()
        combined._js = self.merge(self._js, other._js)
        combined._css = {
            medium: self.merge(self._css.get(medium, []), other._css.get(medium, []))
            for medium in self._css.keys() | other._css.keys()
        }
        return combined
```

Thus, the `dest` object that was passed to the function is not changed here, while that was the whole point of the `add_media` performance boost.

Reply:
However, that was easy to fix, so I've done that in 9d651a1cdb92d7d8ba039fce97a11de085b54990 :-)

Reply:
It took me far too long to understand why it didn't work. Thanks for fixing that! `:)`